### ISSUE TITLE
Error during user defined Project Configuration for custom multi-animal tracking

### ISSUE THREAD

**Issue by drish-soneja** [sgoldenlab/simba#346]
**Describe the bug**
In the define pose window: during assigning body parts to individual animals on the image, the screen has the selected image and there seem to be buttons above the image but they are not visible. 
After assigning the body parts to the animals the define pose window closes and the project configuration page still has different tabs but they are all blank. The main window shows that user defined pose configuration has been created. But since the rest of the tabs on the project configuration page are blank I cannot finish creating the project. 

**To Reproduce**
1. Go to File -> create new project
2. select multi animal tracking and custom tracking (2 animals and 6 body parts per animal)
3. assign user defined nodes to the selected image

**Screenshots**
Terminal:
![Pasted image 3](https://github.com/sgoldenlab/simba/assets/162177197/96c559a6-bcb7-4798-89a2-107a126958e8)

Define pose page during node assignment:
![Pasted image](https://github.com/sgoldenlab/simba/assets/162177197/1a38f81e-fe40-4d74-9bdd-3ce653708c0f)

Blank project configuration page after node assignment
![Pasted image 1](https://github.com/sgoldenlab/simba/assets/162177197/3a4ce731-ef10-4470-96c5-f453bb485c2f)

Main simba window after node assignment:
![Pasted image 2](https://github.com/sgoldenlab/simba/assets/162177197/6866e23d-f31d-47f4-833e-4c06a31374f1)

**Desktop (please complete the following information):**
 - Fedora Linux 38
 - Python Version 3.6.0
 - Are you using anaconda? Yes, I created an environment using conda and then installed simba using pip install simba-uw-tf-dev command

**Comment by sronilsson** [comment]
Thanks for reporting @drish-soneja - can you upgrade to 1.87.1 with `pip install simba-uw-tf-dev --upgrade` and let me know how it looks?

**Comment by drish-soneja** [comment]
hi @sronilsson thank you for your help, the update fixed the problem I was having! 

I did run into another issue when I was trying to annotate my video. None of the key shortcuts were working to navigate the video. 
![Pasted image 1](https://github.com/sgoldenlab/simba/assets/162177197/ffbd0749-cdca-48f8-b2d7-6281d533a89c)
Let me know if I should open a new issue for this.

**Comment by sronilsson** [comment]
Thanks for letting me know @drish-soneja, no need to open another issue. I will see if I can replicate it and get back to you

**Comment by sronilsson** [comment]
@drish-soneja I tried on MacOS and in Ubuntu and unfortunately I can't replicate the error and I don't have a machine running fedora around. The keyboard bindings are controlled though opencv, perhaps it is a opencv version issue - which version of opencv do you have in your simba conda environment? This is what I have. 

![image](https://github.com/sgoldenlab/simba/assets/34761092/8ee14355-caf4-4e62-b882-f37829b66983)

**Comment by drish-soneja** [comment]
I have the same version as you do. Do you think I should update opencv?
![Pasted image 2](https://github.com/sgoldenlab/simba/assets/162177197/686ae02b-3591-4df1-9861-d45a45158e5c)

**Comment by sronilsson** [comment]
No that looks OK.. one more question. Which version of tkiinter does your environment have? 

<img width="427" alt="image" src="https://github.com/sgoldenlab/simba/assets/34761092/12632721-e9a6-41fb-8b77-a0d0736a3c89">

**Comment by drish-soneja** [comment]
I have the same version as you for tkinter too
![Pasted image 3](https://github.com/sgoldenlab/simba/assets/162177197/80cd7495-8cc1-4b7e-ba33-8ebcf2231ab8)

**Comment by drish-soneja** [comment]
the keybinds worked for labelling the body parts when I was defining the pose. I was able the use the x and c keys to change the frame and continue. So I think it might be something specific to this window?

**Comment by sronilsson** [comment]
Yeah - sorry for mentioning opencv thing earlier, it's not actually opencv that binds the keys while labelling, its through tkinter right [here](https://github.com/sgoldenlab/simba/blob/163bd746258fb82b773a60d7a759c6cbe48d33f1/simba/labelling/labelling_interface.py#L326) in the code.  The x and z keys used earlier, is through opencv. 

My suspicion, as I don't see this error on mac or ubuntu, is that it has to do with with the tkinter version running on fedora. One more thing you could help me try to see if this is the cause: you could try to run simba in python3.9 or python3.10 conda environment (I know there will be some warnings but it should run), that gives you a different tkinter version automatically through the python3.9 or python 3.10 standard library, and you could see  if the error persist? Also make sur eyou don't have caps lock pressed.

**Comment by drish-soneja** [comment]
For python = 3.6 

I realised that the frame navigation keybinds (ctl+a,  right, left, etc) are working, its only the video navigation(p, o, etc) that wasnt working. So I can use the frame navigation keybinds to go through my video.

Also when I try to draw an ROI this is what the screen looks like: 
![Pasted image 4](https://github.com/sgoldenlab/simba/assets/162177197/8ec12e3c-1cff-4f86-b785-43305873ebaf)
I'm not able to see frames from my videos but the video information is dispayed in the ROI settings window. 


For python = 3.9
I got a tkinter error when trying to load the project. 
![Pasted image](https://github.com/sgoldenlab/simba/assets/162177197/44254c50-fec5-416d-bc42-3136f0003a82)

also thank you so much for all your help!

**Comment by sronilsson** [comment]
Thank you, super helpful - one question about the ROI issue before I dig in those issues: do you see any error message in the Linux terminal when you "top level not responding" pop up?
